RegistrationApp/RegistrationApp/SignInControllers/SignInViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  SignInViewController.swift
+//  RegistrationApp
+//
+//  Created by Apple Macbook Pro 13 on 29.07.22.
+//
+
+import UIKit
+
+final class SignInViewController: UIViewController {
+    
+    //MARK: IBOutlets
+    @IBOutlet private var emailTextField: UITextField!
+    @IBOutlet private var wrongEmailLbl: UILabel!
+    @IBOutlet private var passTextField: UITextField!
+    @IBOutlet private var wrongPassLbl: UILabel!
+    @IBOutlet private var showPassLbl: UIButton!
+    @IBOutlet private weak var signInButtonOutlet: UIButton!
```

Comment:
why this button is weak, not previous one is not?

Reply:
I don't  know, but I have fixed it